src/web_app/js/util.js

```diff
@@ -74,32 +74,21 @@ function sendUrlRequest(method, url, async, body) {
   });
 }
 
-// Returns a list of turn servers after requesting it from CEOD.
+// Returns a list of turn servers after requesting it from the TURN server.
 function requestTurnServers(turnRequestUrl, turnTransports) {
```

Comment:
It might be useful to put a sample response in a comment above requestTurnServers.